src/main/resources/config/application-docker.yml

```diff
@@ -7,6 +7,14 @@ artemis:
     submission-export-path: /opt/artemis/data/exports
     legal-path: /opt/artemis/data/legal
 
+    version-control:
+        local-vcs-repo-path: /opt/artemis/data/local-vcs-repos
+
+    continuous-integration:
+        local-cis-build-scripts-path: /opt/artemis/data/local-ci-scripts
+        # The Docker connection URI to be used by docker-java to connect to the system's Docker installation (Windows)
+        docker-connection-uri: tcp://host.docker.internal:2375
```

Comment:
This breaks docker deployments on Linux/macOS. Could you try to find a better solution here?

Reply:
I changed this to `unix:///var/run/docker.sock`. That should work for unix systems. I also added some documentation how this can be overwritten if you want to run the containers locally on Windows.